src/functions/socialFunctions.js

```diff
@@ -40,7 +40,7 @@ function socialVolume_ (slug, from, to, socialVolumeType) {
 }
 
 function trendingWords_ (source, size, hour, from, to) {
-  assertCanAccessHistoricData_(from)
+  assertCanAccessHistoricData_(from, source)
```

Comment:
Why is this `source` here? Shouldn't it be `slug`? I guess the problem is that this function here is not working with slugs, so something needs to be passed to it. I think it makes more sense to pass `nil` and indicate an empty slug.